docs/02-app/01-building-your-application/11-upgrading/06-from-create-react-app.mdx

```diff
@@ -1,585 +0,0 @@
----
-title: Migrating from Create React App
```

Comment:
Curious why we are removing this page?

Reply:
It's duplicated, I think it was a bad merge.